source/_integrations/music_assistant.markdown

```
@@ -20,6 +20,7 @@ The **Music Assistant** (MA) {% term integration %} allows you to connect Home A
 There is currently support for the following Home Assistant Platforms:
 
 - [Media player](#media-player)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)